src/main/java/com/cisco/trex/stateful/model/stats/HistogramData.java

```diff
@@ -0,0 +1,29 @@
+package com.cisco.trex.stateful.model.stats;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class HistogramData {
```

Comment:
can make it inner class of LatencyPortHist and rename it with "histogram", then all names can keep alignment at most with original json format

Reply:
good suggestion, thanks.